src/submission/models.py

```diff
@@ -768,10 +773,20 @@ def get_pubid(self):
         return self.get_identifier('pubid')
 
     def is_accepted(self):
-        # return true for all stages after accepted
-        return self.stage == "Published" or self.stage == "Accepted" or self.stage == "Editor Copyediting"\
-            or self.stage == "Author Copyediting" or self.stage == "Final Copyediting"\
-            or self.stage == "Typesetting" or self.stage == "Proofing"
+        from core import models as core_models
+        if self.date_published:
+            return True
+
+        if ArticleStageLog.objects.filter(
+            article=self,
+            stage_to='Accepted',
+        ).exists():
+            return True
+
+        if not self.journal.element_in_workflow('review') and self.stage not in NEW_ARTICLE_STAGES:
+            return True
```

Comment:
I hadn't thought of considering a  `NEW_ARTICLE_STAGES`, maybe now we can simplify the whole method to:

`self.stage not in (NEW_ARTICLE_STAGES + REVIEW_STAGES)`?

Reply:
NOICE.